### PR TITLE
Use valid version in maintainer guide

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by ricknox** [comment]
Looks good to me √

**Review by ghost — APPROVED** [review]
√

**Review by javiereguiluz — APPROVED** [review]
I like this attention to detail ... but this document is for internal use only, so it's not critical to keep it updated like this. In any case, thanks for this contribution Oskar.

File: _build/maintainer_guide.rst [hunk 1]
```diff
@@ -195,7 +195,7 @@ Step 3: Merge it into the other branches
 
 If a PR has not been merged in ``master``, you must merge it up into all the
 maintained branches until ``master``. Imagine that you are merging a PR against
-``3.4`` and the maintained branches are ``3.4``, ``4.2`` and ``master``:
+``3.4`` and the maintained branches are ``3.4``, ``4.4`` and ``master``:
```
Comment (HeahDude) [inline _build/maintainer_guide.rst]: I would add 5.0 though

Comment (OskarStark) [reply _build/maintainer_guide.rst]: I do this while upmerging 👍🏻

**Pull request merged**